pkgs/development/interpreters/bqn/dzaima-bqn/default.nix

```diff
@@ -0,0 +1,72 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, jdk
+, makeWrapper
+, buildNativeImage ? true
+}:
+
+stdenv.mkDerivation {
+  pname = "dbqn" + lib.optionalString buildNativeImage "-native";
+  version = "0.0.0+unstable=2021-10-02";
+
+  src = fetchFromGitHub {
+    owner = "dzaima";
+    repo = "BQN";
+    rev = "d6bd66d26a89b8e9f956ec4f6b6bc5dcb5861a09";
+    hash = "sha256-BLRep7OGHfDFowIAsBS19PTzgIhrdKMnO2JSjKuwGYo=";
+  };
+
+  nativeBuildInputs = [
+    jdk
```

Comment:
If it is used in makeWrapper it does not need to be added here and if used at runtime it should be buildInputs.

Reply:
`native-image` is not shown in PATH if it is not added to this list.